src/widgets/table.rs

```diff
@@ -49,6 +49,13 @@ impl TableState {
         );
     }
 
+    pub fn set_rows(&mut self, rows: Vec<Vec<&str>>) {
```

Comment:
Could you add a doc line here please ?

Reply:
Done! Added doc comment with an extra note.